spectrum/src/main/res/values/attrs.xml

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <attr name="spectrum_colors" format="reference" />
+    <attr name="spectrum_strokeWidth" format="dimension" />
```

Comment:
That is a good improvement, I was unsure about the name my self. 
borderStrokeWidth sounds better